src/home/components/Notification/Notification.tsx

```diff
@@ -12,24 +14,23 @@ import {
   StyledNotificationContainer,
 } from './Notification.style';
 
-const Dummy = [
-  '그거 아시나요?? 지금은 2월이라는 거? 당연함 2월임',
-  '안녕하세요~~ 공지입니다.',
-  '가나다라마바사아자차카타파하갸냐댜랴먀뱌샤야쟈챠캬탸퍄햐거너더러머버서어저처커터퍼허겨녀뎌려며벼셔여져쳐켜텨펴혀',
-  '이거슨 테스트여',
-];
-
-// TODO: 추후 공지사항 API 연결 확인 필요
-export const Notification = () => {
+const NotificationContent = () => {
+  const { data } = useGetAnnouncement();
   const [currentIndex, setCurrentIndex] = useState(1);
```

Comment:
아래와 같은 이유로 index도 0부터 시작해도 될 것 같습니다!
```suggestion
  const [currentIndex, setCurrentIndex] = useState(0);
```

Reply:
여기에서 수정했습니다!

[refactor: 배열 추가 방법을 수정하여 무한 슬라이드 로직 단순화](https://github.com/yourssu/Soomsil-Web/pull/205/commits/a75b4e43a73cb21df2191e51330b802462b3f78f)